tests/integration/conftest.py

```diff
@@ -349,14 +362,14 @@ def __call__(
 
 
 @pytest.fixture
-async def run_actor(apify_client_async: ApifyClientAsync) -> RunActorFunction:
+async def run_actor(apify_client_async: ApifyClientAsync, run_input: Any = None) -> RunActorFunction:
```

Comment:
Does this let you make a per-module fixture with the run input or am I reading it wrong? Also, is it used anywhere?

Reply:
whoops, that should not be there, good catch, thanks